### ISSUE TITLE
Songs that are 'not downloadable'.

### ISSUE THREAD

**Issue by tyhallcsu** [Miserlou/SoundScrape#74]
`8Er$ x Kimfu - Everybody Got Time For That (Original Mix) is not downloadable.`

Sound found at:
https://soundcloud.com/capslockcrew/8er-x-kimfu-everybody-got-time-for-that-original-mix

Why won't the script download this particular track?
I can download it through DownloadHelper, so not a big deal, just wondering why this particular track is being prevented from downloading through soundscrape...?

**Comment by tyhallcsu** [comment]
Another track:

```
Downloading: CloZee - Get Up Now
[################################] 4189/4189 - 00:00:02
'ascii' codec can't encode character u'\u016b' in position 4: ordinal not in range(128)
```

Track found at:
https://soundcloud.com/otodayosupa/clozee-get-up-now

What is this error telling us?

**Comment by Miserlou** [comment]
Thanks for reporting.

There are two different errors going on here. The first is trickier - the second I can probably fix in a second, it's just a unicode thing. Sec..

**Comment by Miserlou** [comment]
Just published 0.23.10 which contains a fix for both of these tracks. Thanks again for reporting.
